cpp/modmesh/transform/fourier.hpp

```diff
@@ -25,15 +29,15 @@ void dft(SimpleArray<T1<T2>> const & in, SimpleArray<T1<T2>> & out)
     {
         for (size_t j = 0; j < N; ++j)
         {
-            T2 tmp = -2.0 * pi<T2> * i * j / N;
+            T2 tmp = -2.0 * pi<T2> * i * j / static_cast<T2>(N);
```

Comment:
Preventing type conversion after dividing a `size_t`, which can cause loss of numerical precision.

Reply:
Good catch